### ISSUE TITLE
Two Bugs

### ISSUE THREAD

**Issue by XiaoWang-Github** [lucidrains/x-transformers#123]
1. In examples/enwik8_simple/train.py ,
Line 71 needs to add drop_last to the Dataloader. So instead of
train_loader  = cycle(DataLoader(train_dataset, batch_size = BATCH_SIZE))  ,
You need to have:
train_loader  = cycle(DataLoader(train_dataset, batch_size = BATCH_SIZE,drop_last=True))

Otherwise, you might get weird result when the length of the dataset, len(train_dataset), doesn't divide the batch size.  Similarly, you need to add the same argument to validate_loader.

2. In line 1217 in x_transformers.py
Instead of :       self.post_emb_norm = nn.LayerNorm(dim) if post_emb_norm else nn.Identity()
It should be:
        self.post_emb_norm = nn.LayerNorm(**emb_dim**) if post_emb_norm else nn.Identity()

```

**Comment by lucidrains** [comment]
@XiaoWang-Github thanks Xiao!
